mne/viz/_3d.py

```diff
@@ -1947,3 +1948,133 @@ def _dipole_changed(event, params):
                  params['idx'], params['dipole'], params['gridx'],
                  params['gridy'], params['ori'], params['coord_frame'],
                  params['zooms'], params['show_all'], params['scatter_points'])
+
+
+def plot_glass_brain(stc, subject, subjects_dir=None, fwd=None, time=0.,
+                     display_mode='lyrz', colorbar=False, axes=None,
+                     title=None, threshold='auto', annotate=True,
+                     black_bg=False, cmap=None, alpha=0.7, vmin=None,
+                     vmax=None, plot_abs=True, symmetric_cbar="auto"):
+    """Plot stc on glass brain.
+
+    Parameters
+    ----------
+    stc : instance of SourceEstimate
+        Data to plot.
+    subject : str
+        The subject name corresponding to FreeSurfer environment
+        variable SUBJECT.
+    subjects_dir : None | str
+        The path to the freesurfer subjects reconstructions.
+        It corresponds to Freesurfer environment variable SUBJECTS_DIR.
+        The default is None.
+    fwd : instance of SourceSpace
```

Comment:
I changed it because I noticed the number of sources didn't match with the data. I suppose it's because some were omitted because they were [too close to the inner skull](https://github.com/mne-tools/mne-python/blob/master/mne/source_space.py#L2108). The omissions are not reflected to ``src``.